src/components/wallet/WalletWidgetDetail/WalletWidgetDetail.jsx

```diff
@@ -217,104 +209,71 @@ export default class WalletWidgetDetail extends PureComponent {
                 </div>
               </div>
               <div styleName='content-container'>
                 <div styleName='address-title'>
                   <h3>{this.getWalletName()}</h3>
-                  <span styleName='address-address'>{address}</span>
-                </div>
-                <div styleName='token-amount'>
-                  <div styleName='crypto-amount'>
-                    {tokensList.length === 1
-                      ? (
-                        <div>
-                          <div>{tokensList[0].symbol} {integerWithDelimiter(tokensList[0].amount.toFixed(2), true)}</div>
-                          <div styleName='amountSubTitle'>USD {integerWithDelimiter(tokensList[0].amountPrice.toFixed(2), true)}</div>
-                        </div>
-                      )
-                      : (
-                        <div>
-                          <div>USD {integerWithDelimiter(walletInfo.balance.toFixed(2), true)}</div>
-                          <div styleName='tokensSubTitle'>
-                            <Translate value={`${prefix}.tokensTitle`} count={tokensList.length} />
-                            {wallet.isMultisig() ? <span>, <Translate value={`${prefix}.ownersTitle`} count={wallet.owners().size()} /></span> : null}
-                          </div>
-                        </div>
-                      )}
-                  </div>
+                  <span styleName='address-address'>{wallet.address}</span>
                 </div>
+                {token && token.isFetched() ? <WalletMainCoinBalance wallet={wallet} /> : <span styleName='noToken'><Translate value={`${prefix}.tokenNotAvailable`} /></span>}
 
-                {wallet.isTimeLocked() && (
+                {wallet.isTimeLocked && (
                   <div styleName='unlockDateWrapper'>
-                    <Translate value={`${prefix}.unlockDate`} /> <Moment data={wallet.releaseTime()} format='HH:mm, Do MMMM YYYY' />
+                    <Translate value={`${prefix}.unlockDate`} /> <Moment data={wallet.releaseTime} format='HH:mm, Do MMMM YYYY' />
                   </div>
                 )}
 
               </div>
             </div>
             <div styleName='footer'>
               <div styleName='pendingOperations'>
-                {wallet.isMultisig() && (
+                {wallet.pendingCount > 0 && (
                   <div styleName='pendingOperationsCount'>
-                    <Translate value={`${prefix}.pending`} count={wallet.pendingTxList().size()} />
+                    <Translate value={`${prefix}.pending`} count={wallet.pendingCount} />
                   </div>
                 )}
               </div>
               <div styleName='actions-container'>
                 <div styleName='action'>
                   <Button
-                    disabled={false}
+                    disabled={!token || !token.isFetched()}
```

Comment:
You may create `const tokenState = (!token || !token.isFetched())` before return and use it here and at line 276:
```
disabled={tokenState}
```